test/CMakeLists.txt

```diff
@@ -214,14 +218,18 @@ add_umf_test(
 
 # tests for the proxy library
 if(UMF_PROXY_LIB_ENABLED AND UMF_BUILD_SHARED_LIBRARY)
+    set(PROXY_LIBS umf_proxy)
+    if(PROXY_LIB_USES_JEMALLOC_POOL)
+        set(PROXY_LIBS ${PROXY_LIBS} ${JEMALLOC_LIBRARIES})
```

Comment:
Why should tests link with jemalloc? Linking with umf_proxy should be enough since it already links with jemalloc.